src/content/en/agencies/js/directory.js

```diff
@@ -0,0 +1,656 @@
+/**
```

Comment:
Eeep! This is going to require a review by someone with JS readability in google3 and an LGTM from them when the CL is created.

@ebidel - can you pre-review this please?